### DISCUSSION TITLE
Is Evidence under active development?

### DISCUSSION THREAD

**Discussion by amit-cashify** [evidence-dev/evidence#2822] [Q&A] [answered]
I see recent releases on GitHub but the https://evidence.dev/changelog page mentions the last releases in September. I am looking to start using Evidence. 

Thanks

**Comment by hughess** [comment, marked as answer]
Yes it is! That changelog is out of date but will be updated today. We release a new version of evidence every week.

More information on releases can be found in our slack community: https://slack.evidence.dev
